src/js/utils/url.js

```diff
@@ -85,6 +85,10 @@ export const parseUrl = function(url) {
     document.body.removeChild(div);
   }
 
+  if (details.protocol === '') {
+    details.protocol = window.location.protocol;
+  }
+
```

Comment:
Overally, seems like a good change/fix. I wonder if it ought to be simply testing `!details.protocol` instead?

Also, maybe it should be moved above the `if (addToBody) {}` block to keep the `details.protocol` conditions together?

Reply:
There should be no protocol that is as well legal as falsy. So ```!details.protocol``` sounds good.

Do you refer only to readability with your second suggestion or should there be a functional difference I don't see?

I guess you see it still below the addToBody declaration, don't you?

Reply:
Yep, exactly. That'd be great, thanks!